server/models/buoy.py

```diff
@@ -1,5 +1,6 @@
-from geoalchemy2.types import Geography  # type: ignore
-from sqlalchemy import Column, Float, Integer, SmallInteger, String  # type: ignore
+from geoalchemy2.types import Geography
+from sqlalchemy import Column, Float, Integer, SmallInteger, String
```

Comment:
Please re-add `# type: ignore` for both these imports. `sqlalchemy` and `geoalchemy2` do not natively provide types to support `mypy`, so we're using this as a temporary measure in order to provide some type checking functionality now.

Reply:
Thing is I didn't touch those lines of code, I think your scripts did. So you might want to investigate why it's doing so.

Reply:
You modified this file when handling the merge conflict.

Reply:
![5D0E5AE3-2417-4BD4-B933-F7F9F08C7390](https://user-images.githubusercontent.com/61562774/185802911-023ab702-da58-4a1f-a760-0823550d3113.png)